script.js

```diff
@@ -1,11 +1,20 @@
 // Create an array called `colors` that contains three colors.
-
+var colors = ["red", "white", "blue"]{
+}
 // Log the last item in the array to the console.
-
+console.log
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
-
+colors.forEach(function(colors) {
+  console.log("My favorite color is []" + colors)
+});
+}
 // Create an `instructor` object that contains three key-value pairs.
+var= instructor ['height', 'class', 'period']{
```

Comment:
for this we'd want to:
```
var instructor = {
  height: 80,
  class: "Web Dev",
  period: 2
}
```
being sure to (1) name the variable immediately following the `var` keyword and before the assignment operator `=`; (2) use curly braces `{}` around the key, value pairs; and (3) provide both keys and values (`key: value`) with commas between pairs